package/MDAnalysis/coordinates/INPCRD.py

```diff
@@ -89,3 +152,116 @@ def parse_n_atoms(filename, **kwargs):
             f.readline()
             n_atoms = int(f.readline().split()[0])
         return n_atoms
+
+
+class NCRSTReader(base.SingleFrameReaderBase, NCDFMixin):
+    """Reader for `AMBER NETCDF format`_ (version 1.0 rev C) restart files.
+
+    This reader is a :class:`SingleFrameReaderBase` adaptation of the
+    :class:`NCDFReader` AMBER NETCDF trajectory reader.
+
+    AMBER binary restart files are automatically recognised by the file
+    extensions ".ncrst", ".ncrestrt", and ".ncrst7".
+
+    The number of atoms (`n_atoms`) does not have to be provided as it can
+    be read from the input NETCDF file.
+
+    Current simulation time is autodetected and if available is read into the
+    :attr:`Timestep.time` attribute.
+
+    Velocities are autodetected and read into the :attr:`Timestep._velocities`
+    attribute.
+
+    Forces are autodetected and read into the :attr:`Timestep._forces`
+    attribute.
+
+    Periodic unit cell information is detected and used to populate the
+    :attr:`Timestep.dimensions` attribute. (If no unit cell is available in
+    the restart file, then :attr:`Timestep.dimensions` will return
+    ``[0,0,0,0,0,0]``).
+
+    Support for the *mmap* keyword is available as detailed
+    in :class:`NCDFReader` and :mod:`scipy.io.netcdf.netcdf_file`. The use of
+    ``mmap=True`` leads to around a 2x read speed improvement in a ~ 1 million
+    atom system (AMBER STMV benchmark). As per the :class:`NCDFReader`, the
+    default behaviour is ``mmap=None``, which means that the default behaviour
+    of :class:`scipy.io.netcdf.netcdf_file` prevails.
+
+    The NCRST reader also uses a custom Timestep object with C-style memory
+    mapping in order to match the NCDFReader.
+
+    .. rubric:: Limitations
+
+    * Only NCRST files with time in ps, lengths in Angstroem and angles in
+      degree are processed.
+    * Restart files without coordinate information are not supported.
+    * Replica exchange variables are not supported.
+
+    .. _AMBER NETCDF format: http://ambermd.org/netcdf/nctraj.xhtml
+
+    See Also
+    --------
+    :class:`NCDFReader`
```

Comment:
Some hyperlinks to classes within this module are not working. You need to use absolute references, for example,
```
:mod:`MDAnalysis.coordinates.INPCRD`
```